### ISSUE TITLE
PWings colliders

### ISSUE THREAD

**Issue by PhineasFreak** [Crzyrndm/ProceduralWings#6]
The dev version of PWings has a problem with the colliders not rendering correctly. Upon selecting a PWing part from the part list (does not matter what the actual PWing part is), a NullRef is logged (sample below):

```
[ERR 10:20:19.218] Module WingManipulator threw during OnStart: System.NullReferenceException: Object reference not set to an instance of an object
  at pWings.WingManipulator.SetupCollider () [0x00000] in <filename unknown>:0 
  at pWings.WingManipulator.Setup (Boolean doInteraction) [0x00000] in <filename unknown>:0 
  at pWings.WingManipulator.OnStart (StartState state) [0x00000] in <filename unknown>:0 
  at Part.ModulesOnStart () [0x00000] in <filename unknown>:0 
```

If applied with symmetry, the parent symmetry part cannot be selected for editing, only it's childs.

After reverting back to SPH/VAB you will not be able to edit the existing PWing parts (cannot be selected).

Tested in a **KSP 1.2.2.1622** installation with only PWings installed.

**Log file: [PW_11032017_104619_output_log.txt](https://raw.githubusercontent.com/PhineasFreak/DocBin/master/DocBin/log_files/PW_11032017_104619_output_log.txt)**

(There is also a weird bug - i think that it has been reported before - where using PWings will force a craft not being able to lift off, even if the TWR is greater than 2, with or without FAR installed. But i will do a proper report once i have tested all the possible mod combinations).

**Edit:** just tested if FAR is able to voxelize the wings but it fails:

```
[EXC 11:16:19.951] NullReferenceException: Object reference not set to an instance of an object
	FerramAerospaceResearch.FARAeroComponents.VehicleAerodynamics.DisplayDebugVoxels (Matrix4x4 localToWorldMatrix)
	FerramAerospaceResearch.FARAeroComponents.VehicleAerodynamics.DebugVisualizeVoxels (Matrix4x4 localToWorldMatrix)
	FerramAerospaceResearch.FARGUI.FAREditorGUI.EditorGUI.DebugVisualizationGUI ()
	FerramAerospaceResearch.FARGUI.FAREditorGUI.EditorGUI.OverallSelectionGUI (Int32 windowId)
	UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID)
	UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style)
```

Also tested the same craft without the wings and everything works as expected.

**Comment by Crzyrndm** [comment]
Should now be fixed with .dll in the repository, but it was compiled/tested against the pre-release (1.2.9.something_or_other). It will probably work in 1.2 release (nothing changed), but no guarantees

**Comment by PhineasFreak** [comment]
Sorry for taking me so long to test and report back...i did so today and i can confirm that the issue is fixed (KSP 1.2.9 pre-release). It crashes under KSP 1.2.2 (at least on a Windows system) but nothing that a recompile cannot fix!
